build/netradiant/install/pkg/default_shaderlist.txt

```diff
@@ -0,0 +1 @@
+common
```

Comment:
in fact this file was not generated for netradiant in initial @neumond work, I guess it's an error and a gtkradiant specific stuff. By the way this file was not pre-populated, but built (copied), but it's hard to guess since git does not know what a copy is.